### PR TITLE
Update @jest/transform dependency to latest 29.* version

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

**Comment by anc95** [comment]
@CR

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/anc95/jest-chain-transform#readme",
   "peerDependencies": {
-    "@jest/transform": "^27.0.6"
+    "@jest/transform": "^29.4.1"
```
Comment (anc95) [inline package.json:28]: could you change this to be ">=27", so that it's Compatible with 27/28/29.. versions

Comment (anc95) [reply package.json:28]: @jdufresne

**Comment by jdufresne** [comment]
Thanks, but I have changed my environment such that I'm no longer using this package.

If you'd like to adapt the change, please go ahead and apply. If not, no problem for me.

Thanks for creating this, it was a huge help when working with babel and TypeScript at the same time.

**Pull request closed without merging**